### PR TITLE
NT-900 - Clear inputfield in modal on focus if zero is the value

### PR DESCRIPTION
**Pull request by eiriko2303**

### CODE REVIEW

File: src/components/time-entry-modal/time-entry-modal.component.ts [hunk 1]
```diff
@@ -232,6 +232,7 @@ export class TimeEntryModal {
     removeZero(event) {
         this.currentInputValue = 0;
         this.currentInputValue = parseFloat(event.target.value);
+        event.target.value == "0" ? event.target.value = '' : null;
```
Comment (reg0) [inline src/components/time-entry-modal/time-entry-modal.component.ts]: please change it to normal if statement for readability

Comment (eiriko2303) [reply src/components/time-entry-modal/time-entry-modal.component.ts]: Ok, change it :) and also added logic for putting 0 if field has no value when focusout :)

**Comment by eiriko2303** [comment]
@reg0 Now it should be better i think ;)

**Comment by reg0** [comment]
@eiriko2303 now the problem is the other way around :) if the input was empty, you go into it, and blur, "0" will appear, although it wasn't there before

**Comment by eiriko2303** [comment]
@reg0 But is that actually a problem? Like when you have a blank field it is 0 so it is the same value?

**Comment by reg0** [comment]
@eiriko2303 ok, you're right, sorry
if you are already in the modal, there always have to be some value

**Pull request merged**